en/docs/design-your-project/explore-the-architecture-view.md

```diff
@@ -0,0 +1,64 @@
+# Explore the Architecture View
+
+Explore the features of the Architecture View using the [Online Boutique microservices demo](https://github.com/ballerina-guides/gcp-microservices-demo) Ballerina project that you set up in the previous section.
```

Comment:
What is meant by previous section?

Reply:
Removed the ` that you set up in the previous section` part.